<launch>
    <remap from="/robot_description" to="/xbotcore/robot_description"/>
    <remap from="/robot_description_semantic" to="/xbotcore/robot_description_semantic"/>

    <node pkg="kyon_controller" type="controller" name="kyon_controller_node" output="screen">
        <param name="config" textfile="$(find kyon_actions)/config/kyon_paw_config.yaml"/>
        <param name="rate" value="1000"/>
    </node>
</launch>